### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by DJ1830** [IceWhaleTech/CasaOS#1298] [Q&A]
I have tried all different variations fo the install command and the system doesn't accept it. Im using Debian 12.1.

**Comment by BaldiJ** [comment]
I think there is an error in the guideline.
On the github guide is reported Debian 12 compatible.
But if you go on the wiki page: https://wiki.casaos.io/en/get-started
you can see there is write Debian 11 compatible 

So I suggest you to try on Debian 11.
I don't know how to report that error on development team...

**Comment by MrGuato** [comment]
Installing CasaOS on Debian 12.1 can be a bit tricky due to compatibility issues. Here’s a step-by-step guide to help you out:

Update Your System: Make sure your system is up-to-date.

```
sudo apt-get update
sudo apt-get upgrade
```
Install Dependencies: CasaOS requires certain dependencies. Install them using the following command:

```
sudo apt-get install -y curl wget gnupg2 software-properties-common
```
Add CasaOS Repository: Add the CasaOS repository to your system.
```
curl -fsSL https://get.casaos.io | sudo bash
```
Install CasaOS: Finally, install CasaOS.
```
sudo apt-get install casaos
